config/app/com/microsoft/azure/iotsolutions/uiconfig/webservice/v1/controllers/PackageController.java

```diff
@@ -85,8 +109,25 @@ public PackageController(IStorage storage) {
 
         final String content = new String(Files.readAllBytes(file.getFile().toPath()));
         final String packageType = data.get(PACKAGE_TYPE_PARAM)[0];
+        String configType = data.get(PACKAGE_CONFIG_TYPE_PARAM)[0];
```

Comment:
will this ever throw an exception? Since you expect edge manifests to not have a config type parameter

Reply:
Isaac sends it empty.

Reply:
ok. it would be safer to check that it exists first in case that changes